hmmlearn/utils.py

```diff
@@ -60,3 +60,9 @@ def iter_from_X_lengths(X, lengths):
 
         for i in range(len(lengths)):
             yield start[i], end[i]
+
+def prob_log(prob):
```

Comment:
a quick check difference is that exp_mask_zero() replaced NaN with Zero while prob_log() replaces -inf with Zero? If this is not the point, I'll take a look at the details of #76.